pkg/engine/k8s/policy/metadata/metadata.go

```diff
@@ -47,7 +47,9 @@ func (m TypeInstanceMetadata) String(withKind bool) string {
 // TypeInstanceIDsWithUnresolvedMetadataForPolicy filters TypeInstances that have unresolved metadata.
 func TypeInstanceIDsWithUnresolvedMetadataForPolicy(in policy.Policy) []TypeInstanceMetadata {
 	var tis []TypeInstanceMetadata
-	for _, rule := range in.Rules {
+
+	// Interface
```

Comment:
Not needed

Reply:
I added that, to show that only one part is currently processed. In next PR I will add support for TypeInstances.

![Screen Shot 2022-02-04 at 09 21 34](https://user-images.githubusercontent.com/17568639/152495598-866d632e-757c-4fda-b998-6e548d65ab3b.png)


I added that as we use  “grouping” comments in other func in this file: https://github.com/capactio/capact/blob/49defde437eb06f6ff249aa055646f52c52836f7/pkg/engine/k8s/policy/metadata/metadata.go#L67-L92